solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -148,13 +151,17 @@ pub mod audius_data {
     }
 }
 
-/// Size of admin account, 8 bytes (anchor prefix) + 32 (PublicKey)
-pub const ADMIN_ACCOUNT_SIZE: usize = 8 + 32;
+/// Size of admin account, 8 bytes (anchor prefix) + 32 (PublicKey) + 8 (id)
```

Comment:
track_id_offset? i think 7 might be enough too

Reply:
u64 is 8 bytes i think 8 is correct